pkg/support/dump_others_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package support
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/utils/exec"
+)
+
+func TestDumpLog(t *testing.T) {
+
+	fs := afero.NewOsFs()
+	basedir := "dir1/dir2"
+	since := "6s"
+
+	varr := NewAgentDumper(fs, nil, nil, nil, nil, since, true, true)
```

Comment:
Use meaningful variable name instead of "varr"
```suggestion
	varr := NewAgentDumper(fs, nil, nil, nil, nil, since, true, true)
```